karma.conf.js

```diff
@@ -27,23 +27,43 @@ module.exports = function(config) {
 
       'jujugui/static/gui/build/app/assets/javascripts/handlebars.runtime.js',
 
-      'jujugui/static/gui/build/app/components/**/*.js',
-      'jujugui/static/gui/build/**/test-*.js'
+      'jujugui/static/gui/src/app/components/**/*.hbs',
+      'jujugui/static/gui/src/app/components/**/*.js'
```

Comment:
This is very opaque could you add a comment above these lines outlining what you just said so that if we ever remove or modify them we know where to look.